styles/hydrogen.less

```diff
@@ -5,487 +5,166 @@
-        }
-
-        &:not(.rich) {
-            .bubble-action-panel {
-                box-shadow: none;
-                display: flex;
-                flex-direction: column;
-                width: 30px;
-                background-color: darken(@bubble-background, 10%);
-            }
-
-            .bubble-output-container {
-                border-radius: 3px 0 0 3px;
-            }
-        }
-
-        .bubble-output-container {
-            display: inline-block;
-            width: 100%;
-            max-width: 100%;
-            max-height: 350px;
-            white-space: pre-wrap;
-        }
-
-
-        .bubble-result-container {
-            max-width: inherit;
-            max-height: inherit;
-
-            position: relative;
-            code {
-                padding: 3px;
-                font-size: inherit;
-                display: block;
-            }
-
-            video, img, svg {
-                width: unset !important;
-                height: unset !important;
-                max-width: 100% !important;
-                max-height: 350px !important;
-            }
-        }
-
-        .bubble-error-container {
-            code {
-                font-size: inherit;
-            }
-        }
-    }
+.multiline-container .toolbar .icon::before {
+  margin: 0;
+  width: 2.5em;
+  line-height: 2.5em;
+  height: 2.5em;
+  cursor: pointer;
 }
 
-.hidden {
-    display: none;
+.multiline-container .toolbar .icon:hover {
+  color: @hydrogen-inline-color;
 }
 
-.hydrogen.input-view {
-    .label {
-        padding-bottom: @component-padding;
-    }
+// -------------- Spinner -----------------
+.spinner {
+  padding: 0 0.5em 0 0.5em;
 }
 
-.hydrogen {
-    .spinner {
-        display:inline-block;
-        text-align: center;
-        line-height: 100%;
-        position: relative;
-        top: 1px;
-    }
+.spinner > div {
+  background-color: @hydrogen-inline-color;
+  height: 100%;
+  width: 2px;
+  display: inline-block;
+  margin-left: 1px;
+  animation: stretchdelay 1.2s infinite ease-in-out;
+}
 
-    .spinner > div {
-        background-color: @text-color;
-        height: 100%;
-        width: 2px;
-        display: inline-block;
+.spinner .rect1 {
+  margin-left: 0px;
+}
 
-        -webkit-animation: stretchdelay 1.2s infinite ease-in-out;
-        animation: stretchdelay 1.2s infinite ease-in-out;
-    }
+.spinner .rect2 {
+  animation-delay: -1.1s;
+}
 
-    .spinner .rect2 {
-        -webkit-animation-delay: -1.1s;
-        animation-delay: -1.1s;
-        margin-left: 1px;
-    }
+.spinner .rect3 {
+  animation-delay: -1.0s;
+}
 
-    .spinner .rect3 {
-        -webkit-animation-delay: -1.0s;
-        animation-delay: -1.0s;
-        margin-left: 1px;
-    }
+.spinner .rect4 {
+  animation-delay: -0.9s;
+}
 
-    .spinner .rect4 {
-        -webkit-animation-delay: -0.9s;
-        animation-delay: -0.9s;
-        margin-left: 1px;
-    }
+.spinner .rect5 {
+  animation-delay: -0.8s;
+}
```

Comment:
You could nest selectors in less (unless we have a reason not to). For example all the spinner styling could be done with only one :spider:

```less
.spinner {
    padding: 0 0.5em 0 0.5em;

    div {
      background-color: @hydrogen-inline-color;
      height: 100%;
      width: 2px;
      display: inline-block;
      margin-left: 1px;
      animation: stretchdelay 1.2s infinite ease-in-out;
    }

    .rect1 {
      margin-left: 0px;
    }

    .rect2 {
      animation-delay: -1.1s;
    }

    .rect3 {
      animation-delay: -1.0s;
    }

    .rect4 {
      animation-delay: -0.9s;
    }

    .rect5 {
      animation-delay: -0.8s;
    }
}

```